### PR TITLE
Formatted code with prettier and used icons instead of .png files

### PR DESCRIPTION
**Pull request by SoumyadiptoPal**
Issue #55
I have formatted the code with prettier. Also used imported icons from 
@expo/vector-icons@13.0.0 instead  of .png images
Updated Screen

### CODE REVIEW

**Comment by mdshamoon** [comment]
@SoumyadiptoPal can you please check the comments in #52 and update those. Also please resolve the merge conflicts

**Comment by mdshamoon** [comment]
@SoumyadiptoPal  let me know once this is ready for review

**Comment by SoumyadiptoPal** [comment]
This code is ready for review. The failed test cases are due to other files

**Comment by SoumyadiptoPal** [comment]
@mdshamoon this is ready for review.

**Comment by mdshamoon** [comment]
@SoumyadiptoPal I have shared some feedbacks on your previous PR and those are not yet resolved here. Can you please check them and update https://github.com/glific/mobile/pull/57#pullrequestreview-1445883685

**Comment by mdshamoon** [comment]
@SoumyadiptoPal any progress on this. Let me know if you need any help

**Comment by SoumyadiptoPal** [comment]
> @SoumyadiptoPal any progress on this. Let me know if you need any help

Thank you very much for asking me. Will me making a pull request soon. Can you please help me to solve the unit test cases issues

**Comment by mdshamoon** [comment]
If you have any specific issue can you come to our discord channel and ask there

**Comment by SoumyadiptoPal** [comment]
@mdshamoon I have updated the code based on your review

File: components/NotificationItem.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <Text style={styles.header}>{notification.header}</Text>
+        <Text>{notification.message}</Text>
+        <Text style={styles.time}>{notification.time}</Text>
+      </View>
+    </View>
+  );
+};
+
+export default NotificationItem;
+const colors = [
```
Comment (mdshamoon) [inline components/NotificationItem.tsx:46]: Please check theme.ts file for colors and add there if not present

**Review by mdshamoon — APPROVED** [review]
Overall looks good. Just update the colors variable and pick it up from them

File: components/NotificationItem.tsx (hunk 1 above)
Comment (SoumyadiptoPal) [reply components/NotificationItem.tsx]: Updated the code

**Comment by mdshamoon** [comment]
Test cases are failing please check using yarn test

**Pull request closed without merging**